src/components/AmountTextField.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { TextField, InputAdornment } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  amountField: {
+    margin: theme.spacing(1),
+    maxWidth: 120,
+  },
+}));
+
+function AmountTextField(props) {
+  const classes = useStyles();
+  const { label, value, onChange, disabled } = props;
+
+  return (
+    <TextField
+      className={classes.amountField}
+      helperText={label}
+      placeholder="Unknown"
+      type="number"
```

Comment:
I imagine if you add `defaultValue=""` here you might be able to get rid of the `|| ''` hack.

Reply:
I thought so too but it still displays the default value `0`

Reply:
Ah that's unfortunate